app/routes/questionnaire.py

```diff
@@ -571,3 +582,18 @@ def request_wants_json():
         best == 'application/json'
         and request.accept_mimetypes[best] > request.accept_mimetypes['text/html']
     )
+
+
+def get_return_to_hub_link(schema, block_type, hub_required_sections):
```

Comment:
I still think these functions shouldn't be in this module.

Reply:
Agreed. As they're specific to question pages they could be moved to `question.py` in `app/views/handlers` and returned in `get_context`.